tyk-oss/templates/NOTES.txt

```diff
@@ -1,21 +1,27 @@
-Thank you for installing Tyk OSS Chart {{ .Chart.Name }}. Your release is named {{ .Release.Name }}. 
-
-At this point, Tyk Gateway is fully installed and should be accessible.
+Thank you for installing Tyk OSS Chart. Your release is named {{ .Release.Name }}.
+At this point, Tyk Gateway {{- if .Values.global.components.pump }} and Tyk Pump are {{- else }} is {{- end }} fully installed and should be accessible.
 
 {{- if index .Values "tyk-gateway" "gateway" "control" "enabled" }}
-
-To quickly test everything is ok, you can port-forward gateway pod to {{ index .Values "tyk-gateway" "gateway" "control" "containerPort" }} on your machine:
-    kubectl port-forward service/gateway-control-svc-{{ template "tyk-gateway.fullname" (index .Subcharts "tyk-gateway")}} {{ index .Values "tyk-gateway" "gateway" "control" "port" }}:{{ index .Values "tyk-gateway" "gateway" "control" "containerPort" }}
-and run `curl localhost:{{ index .Values "tyk-gateway" "gateway" "control" "containerPort" }}/hello` to see some data back that includes redis info.
+{{ $gwSvcName := printf "gateway-control-svc-%v" (include "tyk-gateway.fullname" (index .Subcharts "tyk-gateway"))  }}
+To quickly test everything is ok, you can port-forward Tyk Gateway pod:
+    kubectl port-forward --namespace {{ .Release.Namespace }} service/{{ $gwSvcName }} {{ index .Values "tyk-gateway" "gateway" "control" "port" }}:{{ index .Values "tyk-gateway" "gateway" "control" "containerPort" }}
+    curl localhost:{{ index .Values "tyk-gateway" "gateway" "control" "port" }}/hello
 {{- else }}
+{{ $gwSvcName := printf "gateway-svc-%v" (include "tyk-gateway.fullname" (index .Subcharts "tyk-gateway"))  }}
+To quickly test everything is ok, you can port-forward Tyk Gateway pod:
+    kubectl port-forward --namespace {{ .Release.Namespace }} service/{{ $gwSvcName }} {{ index .Values "global" "servicePorts" "gateway" }}:{{ index .Values "global" "servicePorts" "gateway" }}
+    curl localhost:{{ index .Values "global" "servicePorts" "gateway" }}/hello
+{{- end }}
 
-To quickly test everything is ok, you can port-forward gateway pod to {{ index .Values "global" "servicePorts" "gateway" }} on your machine :
-    kubectl port-forward service/gateway-svc-{{ template "tyk-gateway.fullname" (index .Subcharts "tyk-gateway")}} {{ index .Values "global" "servicePorts" "gateway" }}:{{ index .Values "global" "servicePorts" "gateway" }}
-and run `curl localhost:{{ index .Values "global" "servicePorts" "gateway" }}/hello` to see some data back that includes redis info.
+=== API Secret (TYK_GW_SECRET) ===
+{{- if not .Values.global.secrets.useSecretName }}
+    export TYK_GW_SECRET={{ .Values.global.secrets.APISecret }}
```

Comment:
We should not print the secret in the notes.
We should give command to get it from k8s secret.

Reply:
updated accordingly, can you please check again?